client/src/pages/Chat.js

```diff
@@ -38,7 +38,7 @@ export default function Chat() {
 
 	useEffect(
 		() => {
-			socketRef.current = io.connect("http://localhost:3002/");
+			socketRef.current = io.connect("http://localhost:3001/");
```

Comment:
You should be able to use `window.location.origin` to avoid issues with CORS and deploying. There's a proxy on the frontend so it will direct that properly

Reply:
Changed to window.location.origin